src/main.rs

```diff
@@ -32,7 +32,7 @@ fn run() -> i32 {
 
     let mut opts = Options::new();
     opts.optopt("i", "id", "Window to capture", "ID");
-    opts.optopt("g", "geometry", "Area to capture", "WxH+X+Y");
+    opts.optflagopt("g", "geometry", "Area to capture", "WxH+X+Y");
```

Comment:
Ok, I see what you did here. This means that `-g` without specifying a region would capture the screen where the cursor is. This isn't good, please add a separate flag instead, and document it properly.